tests/simplex/fe_p_bubbles_01.cc

```diff
@@ -58,7 +58,7 @@ test_unit_support_points()
 {
   deallog << "Test support points for dim = " << dim
           << " and spacedim = " << spacedim << std::endl;
-  for (unsigned int degree = 0; degree < 3; ++degree)
+  for (unsigned int degree = 1; degree < 3; ++degree)
```

Comment:
Why isn't degree 0 tested anymore?

Reply:
The element conformity is `H1` so it doesn't make sense to allow piecewise constants.